.github/workflows/codspeed.yml

```diff
@@ -39,19 +39,21 @@ jobs:
           # This is needed because the build-install/_flapackXXX.so file
           # does not link to scipy_openblas32
           echo ">>> manually copy the .so from build -> build-install"
-          ### cp build/openblas_wrap/*.so build-install/lib/python${{matrix.pyver}}/site-packages/openblas_wrap
+          cp build/openblas_wrap/*.so build-install/lib/python${{matrix.pyver}}/site-packages/openblas_wrap
```

Comment:
This `cp` line is to avoid having to set LD_LIBRARY_PATH: it's either copy the library _or_ set LD_LIBRARY_PATH. If you copy it, the library is found at runtime and LD_LIBRARY_PATH is not needed.